lib/email_record.js

```diff
@@ -20,16 +19,14 @@ module.exports = function (limits, now) {
     rec.bk = object.bk       // timestamp when the account was banned
     rec.rl = object.rl       // timestamp when the account was rate-limited
     rec.xs = object.xs || [] // timestamps when emails were sent
-    rec.lf = object.lf || [] // timestamps when a login failure occurred
```

Comment:
I was unsure whether we would want to keep `lf` around for email captcha, but figured we could re-add it with minimal fuss.